providers/image.rb

```diff
@@ -236,10 +253,9 @@ def remove
 
 def repository_and_tag_args
   docker_cmd_args = ''
-  if new_resource.repository
-    docker_cmd_args = new_resource.repository
-    docker_cmd_args += ":#{new_resource.tag}" if new_resource.tag
-  end
+  docker_cmd_args += new_resource.repository + '/' if new_resource.repository
+  docker_cmd_args += new_resource.image_name.split(':', 2).first # strip tag if any
```

Comment:
Should this functionality exist in `image_and_tag_arg` instead? Do we need to remove `repository_and_tag_args` in favor of `repository_image_and_tag_arg`?